src/browser-service.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * MIT License
+ *
```

Comment:
can you move this file to `src/core`

Reply:
I don't think the browser service belongs in the core, since it's not a component that we use directly but rather a service that is executed on it's own. In that sense it's more similar to the cli.